warehouse/models/intermediate/gtfs/gtfs_quality/_int_gtfs_quality.yml

```diff
@@ -347,3 +351,77 @@ models:
             - organization_key
       - dbt_utils.equal_rowcount:
           compare_model: ref('int_gtfs_quality__organization_guideline_index')
+    columns: *stg_gtfs_guideline_columns
+
+# --- Dataset-level
+# ----- Schedule only
+
+  - name: int_gtfs_quality__shapes_accurate
+    tests: &stg_gtfs_guideline_tests_dataset_schedule
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - date
+            - gtfs_dataset_key
+      - dbt_utils.equal_rowcount:
+          compare_model: ref('int_gtfs_quality__gtfs_dataset_schedule_guideline_index')
+    columns: *stg_gtfs_guideline_columns
+
+  - name: int_gtfs_quality__data_license
+    tests: *stg_gtfs_guideline_tests_dataset_schedule
+    columns: *stg_gtfs_guideline_columns
+
+# ----- All 4 feeds
+
+  - name: int_gtfs_quality__authentication_acceptable
+    tests: &stg_gtfs_guideline_tests_dataset
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - date
+            - gtfs_dataset_key
+      - dbt_utils.equal_rowcount:
+          compare_model: ref('int_gtfs_quality__gtfs_dataset_guideline_index')
+    columns: *stg_gtfs_guideline_columns
+
+  - name: int_gtfs_quality__stable_url
+    tests: *stg_gtfs_guideline_tests_dataset
+    columns: *stg_gtfs_guideline_columns
+
+  - name: int_gtfs_quality__grading_scheme_v1
```

Comment:
Does this actually apply to RT feeds? I think you have a mismatch here maybe in terms of the `# ----- All 4 feeds` comment and how things are actually implemented below in terms of which index they use

Reply:
regarding the grading scheme, you're right! just switched it to use the above "schedule only" index table.

Regarding the rest, I'm not totally sure what you mean: I think those checks do apply to all 4 feeds. But surely I could be missing something.

Reply:
Sorry I just meant "below" like "in the files themselves" referring to GH diff display order 😓 But yeah I think this was the one that stood out to me